govcd/api_vcd_test.go

```diff
@@ -107,6 +107,8 @@ type TestConfig struct {
 			Description            string `yaml:"description,omitempty"`
 			CatalogItem            string `yaml:"catalogItem,omitempty"`
 			CatalogItemDescription string `yaml:"catalogItemDescription,omitempty"`
+			CatalogItemWithMultiVm string `yaml:"catalogItemWithMultiVm,omitempty"`
+			VmNameInCatalogItem    string `yaml:"vmNameInCatalogItem,omitempty"`
```

Comment:
From the specific VM name one can think it's referring to `CatalogItem` not `CatalogItemWithMultiVm`, hence suggesting to clarify the var name, for example:
`VmNameInCatalogItem` -> `VmNameInMultiVmItem`

Unless, the idea is to use this variable for the simple `CatalogItem` too?

Reply:
renamed